.github/workflows/linux_sanitizers.yml

```diff
@@ -223,6 +223,7 @@ jobs:
       DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
       INSTALL_DIR: ${{ github.workspace }}/install
       INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+      TBB_ENABLE_SANITIZERS: 1
       CC: clang
       CXX: clang++
```

Comment:
BTW, @akashchi do we need compilers here?

Reply:
IIRC, yes as Sanitizer uses it/it is its dependency and is installed at https://github.com/openvinotoolkit/openvino/pull/26278/files#diff-ac3f8d84a8e20a72d9234847a55d421dc4ef7e838fdeee3bb02a77599e9350c0R267.